packages/babel-plugin-remove-graphql-queries/src/index.js

```diff
@@ -340,39 +367,37 @@ export default function({ types: t }) {
         // Traverse once again for useStaticQuery instances
         path.traverse({
           CallExpression(hookPath) {
+            if (!isUseStaticQuery(hookPath)) return
+
+            // Also see if it's a variable that's passed in as a prop
+            // and if it is, go find it.
             if (
-              hookPath.node.callee.name !== `useStaticQuery` ||
-              !hookPath.get(`callee`).referencesImport(`gatsby`)
+              hookPath.node.callee.name !== `graphql` &&
+              hookPath.node.arguments.length === 1
             ) {
```

Comment:
This heuristic seems to make some non-obvious assumptions. A code example in the comment describing what this is expected to find would be nice. That said, it looks like you migrated that assumption so not holding that against you in this PR.